openedx/adg/lms/webinars/models.py

```diff
@@ -219,6 +219,12 @@ def is_delivered_webinar(self):
     def end_time(self):
         return self.start_time + self.duration
 
+    @property
+    def webinar_id(self):
+        if self.meeting_link:
+            return self.meeting_link.split('/')[-1]
```

Comment:
Are we talking about zoom generated meeting_link?
The name `webinar_id` in this context is a little confusing. If it is the zoom id, then can we name it `zoom_id`?